spec/models/medical_safety_alert_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'spec_helper'
+
+describe MedicalSafetyAlert do
+
+  def medical_safety_alert_content_item(n)
+    {
+      "content_id" => SecureRandom.uuid,
+      "base_path" => "/drug-device-alerts/example-medical-safety-alert-#{n}",
+      "title" => "Example Medical Safety Alert #{n}",
+      "description" => "This is the summary of example Medical Safety Alert #{n}",
+      "format" => "specialist_document",
+      "publishing_app" => "specialist-publisher",
+      "rendering_app" => "specialist-frontend",
+      "locale" => "en",
+      "phase" => "live",
+      "public_updated_at" => "2015-11-16T11:53:30",
+      "publication_state" => "draft",
+      "details" => {
+        "body" => "## Header" + ("\r\n\r\nThis is the long body of an example Medical Safety Alert" * 10),
+        "metadata" => {
+          "alert_type" => "company-led-drugs",
+          "issued_date" => "2016-02-01",
+          "document_type" => "medical_safety_alert"
+        },
+        "change_history" => [],
+      },
+      "routes" => [
+        {
+          "path" => "/drug-device-alerts/example-medical-safety-alert-#{n}",
+          "type" => "exact",
+        }
+      ],
+      "redirects" => [],
+      "update_type" => "major",
+    }
+  end
+
+  let(:non_medical_safety_alert_content_item) {
+    {
+      "content_id" => SecureRandom.uuid,
+      "base_path" => "/other-documents/not-a-medical-safety-alert",
+      "format" => "specialist_document",
+      "details" => {
+        "metadata" => {
+          "document_type" => "not_an_medical_safety_alert",
+        },
+      },
+    }
+  }
+
+  let(:mhra_org_content_item) {
+    {
+      "base_path" => "/government/organisations/medicines-and-healthcare-products-regulatory-agency",
+      "content_id" => "240f72bd-9a4d-4f39-94d9-77235cadde8e",
+      "title" => "Medicines and Healthcare products Regulatory Agency",
+      "format" => "placeholder_organisation",
+      "need_ids" => [],
+      "locale" => "en",
+      "updated_at" => "2015-12-01T15:51:43.521Z",
+      "public_updated_at" => "2015-06-29T14:33:29.000+00:00",
+      "phase" => "live",
+      "analytics_identifier" => "EA63",
```

Comment:
This was copied from the existing response. Only the `base_path`, `content_id` and `title` are required.